docs/swagger.yaml

```diff
@@ -423,6 +423,11 @@ definitions:
       wiki_url:
         type: string
     type: object
+  httphandlers.AnalysisStatus:
```

Comment:
If the Swagger spec is being generated from Go annotations, is it necessary to check this file in?

Reply:
I don't think it hurts anything to have it checked in, plus it has some uses if we want to look at it after generation/deployment. I'll remove it from .gitignore.